rest_framework_nested/viewsets.py

```diff
@@ -60,6 +60,9 @@ def initialize_request(self, request, *args, **kwargs):
         Adds the parent params from URL inside the children data available
         """
         request = super().initialize_request(request, *args, **kwargs)
+        
+        if getattr(self, 'swagger_fake_view', False):
+            return request
```

Comment:
Interesting that swagger needs not the code below this.

Reply:
Can you please link the reference of swagger to what this is needed? On a code comment or at least in this PR please.

Reply:
This should explain why it is needed
https://drf-yasg.readthedocs.io/en/stable/openapi.html#a-note-on-limitations

> the request the view sees will always be the request made against the schema view endpoint - e.g. GET /swagger.yaml